data/formats-data.js

```diff
@@ -4561,7 +4568,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	pidove: {
-		randomBattleMoves: ["pluck", "uturn", "return", "detect", "roost", "wish"],
+		randomBattleMoves: ["aircutter", "uturn", "return", "nightslash", "tailwind"],
```

Comment:
pluck has higher accuracy, and pidove has higher atk than spa

Reply:
the point here is z tailwind and scope lens/superluck into crit moves